kitura.js

```diff
@@ -6,11 +6,13 @@ const program = require('commander');
 program
     .version(pjson.version)
     .description('Kitura command-line interface')
-    .command('build', 'build the project in a local container')
+    .command('init', 'scaffold a bare-bones Kitura project')
```

Comment:
I'm not sure that "bare-bones" is the best way to describe this now.

Reply:
No, bare-bones is not the best description. 

What would you prefer? Cloud ready?